ungleich/dynamicweb
# Task/3711/Show all IPv4s and IPv6s in the VM

- Updated get_ipv4 and get_ipv6 for multiple nics. These functions now return comma separated values of ipv4s and ipv6s respectively.
- Reformatted opennebula_api/serializers.py code

To test this PR: 
- If you already have a VM created, then simply visit the /my-virtual-machines page, otherwise create a VM and verify that the listed VM has values in IPv4 and IPv6 fields
